### DISCUSSION TITLE
Great mod !

### DISCUSSION THREAD

**Discussion by thankserwan** [Woodi-dev/Among-Us-Sheriff-Mod#12] [Q&A] [answered]
Hi there !

You've done a tremendous job here, we tested it yesterday and it was GREAT !

I was wondering if you were planning on creating other mods like a jester role for instance ? 

Thanks 👍

**Comment by Woodi-dev** [comment, marked as answer]
I am planning to create other mods yes. There will be some exclusive mods for YouTubers and some for the public community :)
